app.py

```diff
@@ -8,27 +8,50 @@
 from flask import Flask, flash, request, redirect, render_template, url_for, send_from_directory
 from werkzeug.utils import secure_filename
 from converters.json2glm import json2glm
+from converters.json2png import json2png
 
 app = Flask(__name__)
 app.config.from_object("settings.config.DevelopmentConfig")
 
+
 def allowed_file(filename):
     name, extension = os.path.splitext(filename)
     extension = extension[1:].lower()
     return extension in app.config['ALLOWED_EXTENSIONS']
 
+
 @app.route('/')
 def index():
     return render_template('home.html')
 
+
 @app.route('/convert')
 def upload_form():
     return render_template('convert.html')
 
+
 @app.route('/', methods=['POST'])
 def upload_file():
     convert_from = request.form["convertFrom"]
     convert_to = request.form["convertTo"]
+# json2png default parameters
+    output_type = 'summary'
+    with_nodes = False
+    resolution = "300"
+    size = "300x200"
+    limit = None
+    if (convert_from == "json" and convert_to == "png"):
```

Comment:
This is exactly the use case that the supported conversion mapping dict was created to prevent. Use it here.

Reply:
This has been fixed